curriculum/service/static/index.html

```
@@ -3,273 +3,276 @@
-      한 번에 맞히니 안 걸리면 더 줄인다.<br>
-      <b>요리사가 되고 싶다</b> 를 넣으면<br>
-      빈 결과가 나온다.
-    </div>
-  </section>
 
-  <section class="col graph">
-    <h2>그래프</h2>
-    <div id="nodes"></div>
-    <div class="log" id="log">이벤트가 여기 흐른다.</div>
+    <div class="total">총 <b id="total">120</b>시간이에요.</div>
+    <button class="go" id="go">학습 경로 만들기</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)